_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Google Summer of Code Students 2020
+---
+
+We are happy to anounce that MDAnalysis is hosting three [GSoC][gsoc]
+students this year -- @hmacdope, @cbouy, and @yuxuanzhuang. This is
+the first year that MDAnalysis has been accepted as its own
+[organization with GSoC][mda-gsoc] and we are grateful to Google for granting us
+three student slots so that we can have _three exciting GSoC
+projects_.
+
+## Hugo MacDermott-Opeskin: Trajectory New Generation: the trajectory format for the future of simulation
+
+<img
+src="https://pbs.twimg.com/profile_images/1140166815919665152/D-x2Tvae_400x400.jpg"
+title="Hugo MacDermott-Opeskin" alt="Hugo MacDermott-Opeskin"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Trajectory storage has always proved problematic for the molecular
+simulation community, as large volumes of data can be generated
+quickly. Traditional trajectory formats suffer from poor portability,
+large file sizes and limited ability to include metadata relevant to
+simulation. The Trajectory New Generation format developed by the
+[Gromacs][] team represents the first trajectory format with small
+file sizes, metadata storage, archive integrity verification and
+user/software signatures. In [this
+project](https://summerofcode.withgoogle.com/projects/#5116604104310784),
+@hmacdope will refactor this code into C++ to provide clarity and
+usability for both the Gromacs project and for other simulation
+packages and analysis tools and is the primary goal of this
+project. Thin FORTRAN and Python layers are also desirable to
+encourage widespread adoption and are a secondary goal of the
+project. An efficient and transferable implementation of the TNG
+format will represent a major step forward for the computational
+molecular sciences community, enabling easy storage and replication of
+simulations.
+
+This project is a collaboration with the [Gromacs][] developer team
+with @acmnpv from Gromacs serving as a co-mentor.
+
+Hugo MacDermott-Opeskin is a PhD student in computational chemistry at
+the Australian National University. His work focuses on studying
+membrane biophysics through molecular dynamics simulations coupled
+with enhanced sampling techniques.  Hugo can be found on github as
+@hmacdope and on twitter as [@hugomacdermott][hmacdope-twitter]. When
+not hard at work Hugo can be found running or mountain biking in the
+Canberra hills.
```

Comment:
should I also mention things about the project on my twitter? I'm not using it much but might be a good reason to use it more

Reply:
Might as well put my twitter here also "@hugomacdermott"

Reply:
@hmacdope, Might be misunderstanding what you're saying here, I think it's there already on line 46?

Reply:
Sorry @IAlibay completely missed it.

Reply:
@acmnpv sure, tweet about it. Retweeting the MDA announcement (when we have it) will also help.